docs: Tweak docs for 0.4.0 (#1553)
PrQL/prql: book/src/queries/syntax.md

select first_name
```

## Expressions

> Many PRQL statements compute a value with an _expression_. Expressions follow
> the normal rules for computer languages. In the example below, note the use of

Reviewer: ```suggestion
PRQL is made up of  _expressions_, like `2 + 3` or `(1 + x) * y)`. In the example below, note the use of
```

WDYT?

Author: Yes, but the second expression needs an opening "("

Then LGTM
